.zuul.yaml

```diff
@@ -0,0 +1,9 @@
+---
+- project:
+    name: packit-service/ogr
+    check:
+      jobs:
+        - noop
```

Comment:
How is this going to help us?

Reply:
see the email, this is the first step to test the connection b/w ogr and sf.io's zuul